src/app/organizer/ItemTable.m.scss

```diff
@@ -239,6 +239,18 @@ $content-cells: 5;
 .perkSelectable {
   cursor: pointer;
 }
+.enhancedArrow {
+  &::before {
+    content: '';
+    display: inline-block;
+    width: 6px;
+    height: 11px;
+    mask-image: url('images/enhancedArrow.svg');
+    background-color: $enhancedYellow;
+    margin-top: 3px;
+    margin-right: 1px;
```

Comment:
```suggestion
    margin-right: 1px;
    margin-left: -7px
```
Could use negative margin here to keep enhanced & non-enhanced traits aligned e.g.

![dim-enhanced-arrow-alignment](https://user-images.githubusercontent.com/17512262/230766168-9b3cd1ca-612a-4e58-bfa8-602ab8b2266f.png)

Reply:
Smart!